src/distilabel/llms/base.py

```diff
@@ -162,21 +172,27 @@ def get_runtime_parameters_info(self) -> List[Dict[str, Any]]:
         runtime_parameters_info = super().get_runtime_parameters_info()
 
         generation_kwargs_info = next(
-            runtime_parameter_info
-            for runtime_parameter_info in runtime_parameters_info
-            if runtime_parameter_info["name"] == "generation_kwargs"
+            (
+                runtime_parameter_info
+                for runtime_parameter_info in runtime_parameters_info
+                if runtime_parameter_info["name"] == "generation_kwargs"
+            ),
+            None,
```

Comment:
If we don't found the `runtime_parameter_info` with the `name == "generation_kwargs"`, then the generation expression `(... for item in list if item == "something")` will be empty and `next` will raise an `StopGeneration` exception. To avoid that, we set `None` as default for that cases.